src/Microsoft.VisualStudio.Editors/MyExtensibility/MyExtensibilityPropPage.vb

```diff
@@ -267,17 +277,28 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
             linkLabelHelp.Name = "linkLabelHelp"
             linkLabelHelp.TabStop = True
             '
-            'listViewExtensions
+            'DataGridViewExtensions
             '
-            listViewExtensions.AutoArrange = False
-            listViewExtensions.Columns.AddRange(New ColumnHeader() {colHeaderExtensionName, colHeaderExtensionVersion, colHeaderExtensionDescription})
-            resources.ApplyResources(listViewExtensions, "listViewExtensions")
-            listViewExtensions.FullRowSelect = True
-            listViewExtensions.HideSelection = False
-            listViewExtensions.Name = "listViewExtensions"
-            listViewExtensions.ShowItemToolTips = True
-            listViewExtensions.UseCompatibleStateImageBehavior = False
-            listViewExtensions.View = View.Details
+            '
+            'DataGridViewExtensions.AutoArrange = False
+            colHeaderExtensionName.HeaderText = "Name"
+            colHeaderExtensionVersion.HeaderText = "Version"
+            colHeaderExtensionDescription.HeaderText = "Description"
```

Comment:
These should come from the `ApplyResources` below.  If they aren't, then they need to start coming from localized strings, not being hard-coded to ENU strings.

Reply:
I had to change the .resx for those resources from .Text to .HeaderText - is that the correct way to do it?